pkg/volume/quobyte/quobyte.go

```diff
@@ -158,6 +158,23 @@ func (plugin *quobytePlugin) newMounterInternal(spec *volume.Spec, pod *api.Pod,
 		return nil, err
 	}
 
+	if pod != nil {
+		// We use the service account if it's not the default service account
+		if len(pod.Spec.ServiceAccountName) != 0 && pod.Spec.ServiceAccountName != "default" {
+			source.User = pod.Spec.ServiceAccountName
+			source.Group = pod.Spec.ServiceAccountName
```

Comment:
> I couldn't find a way to get the Kubernetes user in the plugin

Kubernetes user info isn't available at this point (actually, I'm not really sure what you mean by that... the user that created the pod? last updated the pod? created the rc that resulted in this pod?)

> We know, actually it's enough to create a Quobyte Volume for the specified User/Group (service account name) for the mapping.

Can you clarify what source.User and source.Group are and how they are used? It looks like you are losing namespace info (so service account "foo" in ns1 and ns2 would appear identical)